specification/app/resource-manager/Microsoft.App/preview/2023-11-02-preview/AppSpaceResources.json

```diff
@@ -0,0 +1,433 @@
+        "operationId": "AppSpaceResources_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "appSpaceName",
+            "in": "path",
+            "description": "Name of the App Space.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "spaceResourceName",
+            "in": "path",
+            "description": "Name of the App Space Resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "spaceResourceEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties of an App Space Resource that need to be updated",
+            "schema": {
+              "$ref": "#/definitions/AppSpaceResource"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "$ref": "#/definitions/AppSpaceResource"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Container App": {
+            "$ref": "./examples/AppSpaceResources_Patch.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeploymentInformation": {
+      "description": "App Space Resource Deployment Information - used to track the deployment information for an App Space Resource's tracked ARM resource",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "ARM Id of the deployment",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Provisioning State for the deployment",
+          "type": "string"
+        }
+      }
+    },
+    "RepositoryInformation": {
+      "description": "App Space Resource Repository Information - used to track the github repository for an App Space Resource's tracked ARM resource",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "url for the repository",
+          "type": "string"
+        },
+        "branch": {
+          "description": "branch in the repository",
+          "type": "string"
+        }
+      }
+    },
+    "Object": {
+      "type": "object",
+      "properties": {}
```

Comment:
why do you need this definition ?

Reply:
I use it for the metadata property bag - see other comment.